src/routes/docs/Sidebar.svelte

```diff
@@ -59,6 +59,13 @@
                     icon: 'icon-send',
                     isParent: true
                 },
+                {
+                    label: 'Sites',
+                    href: '/docs/products/sites',
+                    icon: 'icon-globe-alt',
+                    isParent: true,
+                    new: true
```

Comment:
lets please add a date until when to show a `New` tag. 30/60 days should be enough. See example -
https://github.com/appwrite/website/blob/main/src/routes/docs/advanced/platform/%2Blayout.svelte#L46